benchmarks/pytorch-lightning/prepare.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
+import os
+import json
+from utils import DataOptions
+from simple_parsing import ArgumentParser
+import shlex
+
+
+def main():
+    # If you need the whole configuration:
+    config = json.loads(os.environ["MILABENCH_CONFIG"])
+    data_directory = os.environ["MILABENCH_DIR_DATA"]
+    argv_dict = config["argv"]
+    argv_str = " ".join(f"--{k} {v}" for k, v in argv_dict.items())
```

Comment:
I don't think that's going to work, `k` should already start with `--`. Also an entry such as `{"--flag": True}` should produce the string `--flag` and not `--flag True`, and if `v` is None it should be omitted.

You can import `milabench.multi._assemble_options` to make a list of option strings (and then you won't need shlex.split). I might move that to `milabench.utils`, but it'll have to do for now.

Reply:
Fixed in d8498f3